internal/drand-cli/control.go

```diff
@@ -322,13 +313,14 @@ func printJSON(w io.Writer, j interface{}) error {
 	return nil
 }
 
+//nolint:unused
 func selfSign(c *cli.Context, l log.Logger) error {
```

Comment:
are you readding the use of this back or can we nuke it?

Reply:
Yes, I am, since we need to update the self-signature to use the new scheme that includes that scheme name in the message, not just the address

Reply:
it's already in use actually, I need to remove that nolint.